packages/react-reconciler/src/__tests__/ReactExpiration-test.js

```diff
@@ -755,10 +755,16 @@ describe('ReactExpiration', () => {
 
       // The update finishes without yielding. But it does not flush the effect.
       await waitFor(['B1'], {
-        additionalLogsAfterAttemptingToYield: ['C1'],
+        additionalLogsAfterAttemptingToYield: gate(
```

Comment:
I'm not sure what the expected semantics of this helper is and whether this change is expected.